lib/api/CFieldDataTyper.cc

```diff
@@ -336,6 +336,11 @@ bool CFieldDataTyper::persistState(core::CDataAdder& persister) {
 }
 
 bool CFieldDataTyper::isPersistenceNeeded(const std::string& description) const {
+    // Pass on the request in case we're chained
+    if (m_OutputHandler.isPersistenceNeeded(description) == true) {
```

Comment:
nit: although we prefer `== false` to `!`, I think `== true` is unnecessary for methods that clearly return a `bool`